fabric/src/test/java/TestRegex.java

```diff
@@ -242,15 +242,12 @@ public void ContainerModel_ABILITY_TREE_PATTERN() {
     @Test
     public void DamageLabelParser_DAMAGE_LABEL_PATTERN() {
         PatternTester p = new PatternTester(DamageLabelParser.class, "DAMAGE_LABEL_PATTERN");
-        p.shouldMatch("§4-13 ❤ ");
-        p.shouldMatch("§4-10 ❤ ");
-        p.shouldMatch("§c-8 ✹ ");
-        p.shouldMatch("§e-30 ✦ ");
-        p.shouldMatch("§2-41 ✤ ");
-        p.shouldMatch("§b-21 ❉ ");
-        p.shouldMatch("§f-32 ❋ ");
-        p.shouldMatch("§c-28 ✹ ");
-        p.shouldMatch("§c-116 ✹ §2-17 ✤ ");
+        p.shouldMatch("§e§l-509 §r§e\uE003 §f§l-398 §r§f\uE000 §c§l-5162 §r§c\uE002 §b§l-386 §r§b\uE004 ");
```

Comment:
Not a critique of this particular code, but the new wynn font unicode sequences are really, really making the code hard to read and understand. :-( I wonder if there is anything we can do about it. 

Like, never use these directly in a string but define constants and use them (not sure it is better), or perhaps do some pre-processing in gradle so we can use one kind of symbols in the source and have them replaces with unicode sequences before compilation. Or introduce a rule that we always have a comment with a readable version next to any string that contains a unicode sequence.

Reply:
Can we have like a special syntax, and parse it? Like "§<wynnfont>HERO§</>" or whatever? If we are worried about the performance increase added to parse string literals, we can add a cache and then we will have a one-time cost (which is negligible) and just a quick hash lookup.

And, contrary, add a developer tool (maybe a /wynntils command?) where we can paste a wynn unicode string and get it back as a string formatted in our special syntax?